sitewit/__init__.py

```diff
@@ -1,4 +1,4 @@
 """Python client for the SiteWit API."""
 
-__version__ = '0.8.0'
+__version__ = '0.9.0'
```

Comment:
The same document `Semantic Versioning` says about that.  And regarding our agreement - I'm not sure if we document it anywhere.  But it sounds like a common sense rule.